.github/workflows/ci.yml

```diff
@@ -9,26 +12,31 @@ on:
         options:
           - CI
           - PROD
```

Comment:
How is this going to work when triggered from cron? Is it going to default to CI?